docs/source/getting-started/content/proc_starting-codeready-containers-with-proxy.adoc

```diff
@@ -0,0 +1,19 @@
+[id="starting-codeready-containers-with-proxy_{context}"]
+= Starting {prod} behind proxy
+
+Proxy can be defined as part of environment variable or with `crc config set` command option as following:
+
+```
+$ crc config set http-proxy http://example.proxy.com:<port>
+$ crc config set https-proxy http://example.proxy.com:<port>
+$ crc config set no-proxy '<multiple-no-proxy-entries-with-comma-separated>'
+```
+
+Once proxy is set by environment variable or by config, `crc` commands will able to use it.
```

Comment:
I would add that if both config and env are set, the values set in crc config have priority over the ones set in the environment. Apart from this, looks good to me.

Reply:
@cfergeau done.